### ISSUE TITLE
GAP crash when trying to write a matrix to file

### ISSUE THREAD

**Issue by ThomasBreuer** [gap-packages/meataxe64#41]
Currently the following crash happens in the master branch.
```
gap> f:= MTX64_FiniteField( 2 );;
gap> m:= MTX64_RandomMat( f, 4, 4 );;
gap> MTX64_WriteMatrix( m, "subdir/file" ); # there is no directory 'subdir'
Cannot open output file subdir/file
```

**Comment by stevelinton** [comment]
To fix this nicely, I really need PR #4015 merged in GAP. In the meantime I'll put some kind of workaround in place.

**Comment by stevelinton** [comment]
SHould be fixed now
